geomstats/geometry/grassmannian.py

```diff
@@ -50,6 +50,16 @@ def belongs(self, point, tolerance=TOLERANCE):
             'The Grassmann `belongs` is not implemented.'
             'It shall test whether p*=p, p^2 = p and rank(p) = k.')
 
+    def is_tangent(self, tangent_vec, point):
```

Comment:
Missing docstrings in these 2 methods.

Reply:
`point` should be called `base_point` and (maybe?) default to `None` if there is an intuitive default case?